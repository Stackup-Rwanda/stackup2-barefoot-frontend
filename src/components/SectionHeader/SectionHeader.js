import React from 'react';
import PropTypes from 'prop-types';
import { makeStyles } from '@material-ui/core/styles';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles({
  title: {
    color: '#484848',
    backgroundColor: '#00a7990f',
    padding: 24,
    fontSize: 14,
    fontFamily: 'Roboto',
    fontWeight: 500,
  },
});

const SectionHeader = ({ title }) => {
  const classes = useStyles();

  return (
    <Typography className={classes.title}>
      {title}
    </Typography>
  );
};

SectionHeader.propTypes = {
  title: PropTypes.string.isRequired,
};

export default SectionHeader;
