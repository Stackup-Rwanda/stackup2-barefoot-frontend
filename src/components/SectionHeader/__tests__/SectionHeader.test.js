/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import SectionHeader from '../SectionHeader';

const title = 'Trip requests';

describe('<SectionHeader/>', () => {
  afterEach(() => cleanup);

  it('Should match the SectionHeader component snapshot', () => {
    const tree = renderer.create(<SectionHeader title={title} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
