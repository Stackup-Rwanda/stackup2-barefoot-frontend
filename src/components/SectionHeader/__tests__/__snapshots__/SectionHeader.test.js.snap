// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionHeader/> Should match the SectionHeader component snapshot 1`] = `
<p
  className="MuiTypography-root makeStyles-title-1 MuiTypography-body1"
>
  Trip requests
</p>
`;
